sdk/src/utils/pdf_utils.rs

```diff
@@ -41,12 +41,13 @@ pub(crate) enum Error {
 
 const C2PA_MIME_TYPE: &str = "application/x-c2pa-manifest-store";
 
+#[cfg_attr(test, mockall::automock)]
 pub(crate) trait C2paPdf: Sized {
     /// Load a PDF from a slice of bytes.
     fn from_bytes(bytes: &[u8]) -> Result<Self, Error>;
 
     /// Save the `C2paPdf` implementation to the provided `writer`.
-    fn save_to<W: Write>(&mut self, writer: &mut W) -> Result<(), std::io::Error>;
+    fn save_to<W: Write + 'static>(&mut self, writer: &mut W) -> Result<(), std::io::Error>;
```

Comment:
Same here: `mockall` requires generics items defined in traits to include `'static` lifetimes. I'm looking for a better solution, but this should work in the short term.